app/measurement/views.py

```diff
@@ -284,7 +285,7 @@ def list(self, request):
         measurements = measurements.filter(metric__in=metrics)
         measurements = measurements.filter(
             starttime__gte=params['starttime']).filter(
-            starttime__lte=params['endtime'])
+            starttime__lte=params['endtime']).order_by('-starttime')
```

Comment:
The archive_measurements command filters by `lt`, not `lte` the 'endtime' (`starttime__gte=period_start, starttime__lt=period_end`), should we keep it consistent?